db/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/db/RepositoryTest.kt

```diff
@@ -186,15 +233,21 @@ class RepositoryTest :
             }
         }
 
-        test("skal _ikke_ oppdatere noen inntektsmeldinger med journalpostId dersom siste allerede har") {
+        test("skal oppdatere inntektsmelding med journalpostId selv om den allerede har dette") {
```

Comment:
Her har vi endret oppførselen litt, hvor vi nå oppdaterer innsendingen med `journalpostId` selv om det skulle befinne en `journalpostId` der for denne innsendingen fra før. Usikker på hvilke scenarioer dette kan skje i, hvis det i det hele tatt kan skje.